data/json/overmap/overmap_mutable/river_cave.json

```
@@ -0,0 +1,70 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "river_cave",
+    "subtype": "mutable",
+    "locations": [ "water" ],
```

```suggestion
    "locations": [ "water", "subterranean_empty" ],
```
You need to define locations for every type of terrain you want the overmap special to "overwrite". In this case you want subterranean_empty so that you can overwrite the soil layer in -1 to create the tunnels.

Think this was the issue you needed help with? Tell me if you need something else

---

That was probably it.  I'll test it tonight. Hopefully that's it!